tlatools/org.lamport.tlatools/javacc/tla+.jj

```diff
@@ -1064,7 +978,7 @@ private boolean correctLevel(Token tok) {
 
 
 PARSER_END(TLAplusParser)
-
+
```

Comment:
Note for self: these apparent no-op whitespace changes are removal of "form-feed" characters (ASCII 0x0C). Form-feed is meant to act like a page break. It has fallen out of fashion, and most text editors (including mine) do not render them well, so I'm in favor of their removal.

Reply:
Aha, so they are actually ASCII!